src/modules/always_on_top/dllmain.cpp

```diff
@@ -0,0 +1,265 @@
+#include "pch.h"
+#include <interface/powertoy_module_interface.h>
+#include <interface/powertoy_system_menu.h>
+#include <common/settings_objects.h>
+#include <set>
+#include "trace.h"
+#include "resource.h"
+
+extern "C" IMAGE_DOS_HEADER __ImageBase;
+
+BOOL APIENTRY DllMain(HMODULE hModule, DWORD  ul_reason_for_call, LPVOID lpReserved) {
+  switch (ul_reason_for_call) {
+  case DLL_PROCESS_ATTACH:
+    Trace::RegisterProvider();
+    break;
+  case DLL_THREAD_ATTACH:
+  case DLL_THREAD_DETACH:
+    break;
+  case DLL_PROCESS_DETACH:
+    Trace::UnregisterProvider();
+    break;
+  }
+  return TRUE;
+}
+
+const static wchar_t* MODULE_NAME = L"AlwaysOnTop";
```

Comment:
The module name for the Settings is `Always On Top`
Use `Always On Top` also in the .rc file.
And talking about the .rc file, look at how is done for the Shortcut Guide, to automatically update the version number and the other fields.